src/main/java/com/minecraftabnormals/abnormals_core/core/AbnormalsCore.java

```diff
@@ -63,6 +64,9 @@
 
 import java.util.Map;
 
+import static com.minecraftabnormals.abnormals_core.core.util.DataUtil.AlternativeDispenseBehavior;
+import static com.minecraftabnormals.abnormals_core.core.util.DataUtil.AlternativeDispenseBehavior.alternativeDispenseBehaviors;
+
```

Comment:
These imports would not even be needed if line 158 was changed to `DataUtil.AlternativeDispenseBehavior.alternativeDispenseBehaviors().forEach(DataUtil.AlternativeDispenseBehavior::register);`

Reply:
True, I just wanted the line to be shorter :P I'll remove the imports now

Reply:
Done.